phases/03-deep-learning-core/05-loss-functions/code/main.py

```
@@ -0,0 +1,262 @@
+import math
+import random
+
+
+def mse(predictions, targets):
+    n = len(predictions)
+    total = 0.0
+    for p, t in zip(predictions, targets):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)